etna/datasets/tsdataset.py

```diff
@@ -278,7 +279,9 @@ def make_future(self, future_steps: int) -> "TSDataset":
 
         future_dataset = df.tail(future_steps).copy(deep=True)
         future_dataset = future_dataset.sort_index(axis=1, level=(0, 1))
-        future_ts = TSDataset(future_dataset, freq=self.freq)
+        future_ts = TSDataset(df=future_dataset, freq=self.freq)
+        future_ts.known_future = self.known_future
```

Comment:
Can't we put this known future into the constructor of TSDataset on the previous line?

Reply:
We can't, because the `_check_known_future` fails with `df_exog=Null`

Reply:
May be a comment should be added about this